src/modules/UI.js

```diff
@@ -0,0 +1,30 @@
+export default class UI {
```

Comment:
Following JavaScript best practices , the class UI should be more descriptive to make the code more readable; You can consider renaming this class to todoUI

Reply:
Thanks for the review  @Lornakaboro , I will do that🙏